xbmc/interfaces/python/PythonInvoker.cpp

```diff
@@ -683,23 +678,22 @@ void CPythonInvoker::onError(const std::string& exceptionType /* = "" */,
   }
 }
 
-void CPythonInvoker::initializeModules(
-    const std::map<std::string, PythonModuleInitialization>& modules)
+void CPythonInvoker::initializeModules(const PythonModule modules[], const int numElements)
 {
-  for (const auto& module : modules)
+  for (int i = 0; i < numElements; i++)
```

Comment:
any reason for removing the range for loop?

Reply:
This construct operates locally and can't be used to transfer the size of the array to the base class.